### PR TITLE
[Feature] Clipboard Feature

### PR DESCRIPTION
**Pull request by elemanhillary**
Adds import from clipboard
Closes [#1748 ](https://github.com/getinsomnia/insomnia/issues/1748)

### CODE REVIEW

File: packages/insomnia-app/app/ui/components/modals/settings-modal.js [hunk 1]
```diff
@@ -53,6 +53,11 @@ class SettingsModal extends PureComponent {
     this.modal.hide();
   }
 
+  _handleClipBoard() {
```
Comment (gschier) [inline packages/insomnia-app/app/ui/components/modals/settings-modal.js]: Should probably rename this everywhere to be ImportClipBoard to match the ImportFile and ImportURI pattern 🙂

File: packages/insomnia-app/app/ui/redux/modules/global.js [hunk 2]
```diff
@@ -181,6 +181,26 @@ export function importFile(workspaceId) {
   };
 }
 
+export function clipBoard(workspaceId) {
```
Comment (gschier) [inline packages/insomnia-app/app/ui/redux/modules/global.js]: Same comment applies here to naming. Should be `importClipBoard`

File: packages/insomnia-app/app/ui/redux/modules/global.js [hunk 3]
```diff
@@ -181,6 +181,26 @@ export function importFile(workspaceId) {
...
+  return async dispatch => {
+    dispatch(loadStart());
+    const schema = electron.clipboard.readText();
```
Comment (gschier) [inline packages/insomnia-app/app/ui/redux/modules/global.js]: Oh nice! Didn't know electron had a clipboard api 👍

Comment (elemanhillary) [reply packages/insomnia-app/app/ui/redux/modules/global.js]: resolved

File: packages/insomnia-app/app/ui/redux/modules/global.js (hunk 2 above)
Comment (elemanhillary) [reply packages/insomnia-app/app/ui/redux/modules/global.js]: resolved

File: packages/insomnia-app/app/ui/components/modals/settings-modal.js (hunk 1 above)
Comment (elemanhillary) [reply packages/insomnia-app/app/ui/components/modals/settings-modal.js]: resolved

**Comment by elemanhillary** [comment]
@gschier it can now be merged

**Comment by elemanhillary** [comment]
@gschier can this be merged

**Pull request merged**